core/modules/qdisp/Executive.h

```diff
@@ -124,12 +126,20 @@ class Executive : public std::enable_shared_from_this<Executive> {
     bool xrdSsiProvision(std::shared_ptr<QueryResource> &jobQueryResource,
                          std::shared_ptr<QueryResource> const& sourceQr);
 
+    std::mutex sumMtx; // TEMPORARY-timing
+    int cancelLockQSEASum{0}; // TEMPORARY-timing
+    int jobQueryQSEASum{0}; // TEMPORARY-timing
+    int addJobQSEASum{0}; // TEMPORARY-timing
+    int trackQSEASum{0}; // TEMPORARY-timing
+    int endQSEASum{0}; // TEMPORARY-timing
```

Comment:
I want it on the pdac, but I don't want this level of logging long term. I'll add a ticket for removing or reducing it after we've gotten some numbers for the pdac. Maybe try for a comparison between map and unordered_map as well.